examples/enable_cors_header_set_issue.rs

```diff
@@ -0,0 +1,22 @@
+#[macro_use] extern crate nickel;
+extern crate hyper;
```

Comment:
In response to #368, we are rexporting hyper, so I think you should be able to remove this line.

Reply:
Ok, I just copied it from the previous example. Should I remove it there too?